DayrSuryan/DSEthiop2.xml

```diff
@@ -548,84 +582,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </decoNote>-\->
                      </binding>
                   </bindingDesc>-->
-                  
+
                   </physDesc>
-                  
+
                             </msPart>
-               
+
                <msPart xml:id="p2">
                   <msIdentifier/>
                   <msContents>
                      <summary/>
-                     
+
                      <msItem xml:id="p2_i1">
                         <locus from="77ra" to="84rb"/>
-                        <title type="complete" ref="LIT2384Taamme"/>
```

Comment:
Was there a reason for deleting this part? I see LIT1968Mashaf present in the manuscript